litellm/llms/prompt_templates/factory.py

```diff
@@ -689,7 +689,7 @@ def anthropic_messages_pt_xml(messages: list):
                     {
                         "type": "text",
                         "text": (
-                            convert_to_anthropic_tool_result(messages[msg_i])
+                            json.dumps(convert_to_anthropic_tool_result(messages[msg_i]))
```

Comment:
This is not correct, this method is used for tool calling using XML as Bedrock does not yet allow tool calling using the new format. When it allows, we can use convert_to_anthropic_tool_result directly, but now we should use convert_to_anthropic_tool_result_xml

https://docs.anthropic.com/claude/docs/legacy-tool-use